data/actions/actionsRouter.js

```diff
@@ -0,0 +1,88 @@
+const express = require("express");
+const db = require("../helpers/actionModel");
+const router = express.Router();
+router.use(express.json());
+
+
+router.get("/", (req, res) => {
+ db.get()
+ .then(actions => {
+   console.log(actions)
+   res.status(200).json(actions);
+ })
+ .catch(err => {
+   res.status(500).json({error: "Cant do that !"}) })
+ });
+ 
+ router.get("/:id",validateActionsId, (req, res) => {
+   const id = req.params.id;
+   db.get(id)
+     .then(proj => {
+       res.status(200).json(proj);
+     })
+     .catch(err => {
+       res.status(500).json({ error: "Invalid Id" });
+     });
+ });
+ 
+ router.post("/:id", (req, res) => {
+   const action = req.body;
+   const {id} = req.params;
+  
+   console.log({...action, project_id: id})
```

Comment:
good use of the spread operator but you might want some conditional check to make sure the action is / has what you are actually looking for. Another middleware??